menas/src/main/scala/za/co/absa/enceladus/menas/WebSecurityConfig.scala

```diff
@@ -137,76 +67,82 @@ class WebSecurityConfig {
 
   @Configuration
   @Order(1)
-  class ApiWebSecurityConfigurationAdapter @Autowired()(restAuthenticationEntyPoint: RestAuthenticationEntryPoint,
-                                                        authenticationSuccessHandler: AuthSuccessHandler,
-                                                        authenticationFailureHandler: AuthenticationFailureHandler,
-                                                        logoutSuccessHandler: LogoutSuccessHandler)
+  class ApiWebSecurityConfigurationAdapter @Autowired() (restAuthenticationEntyPoint: RestAuthenticationEntryPoint,
```

Comment:
```suggestion
  class ApiWebSecurityConfigurationAdapter @Autowired() (restAuthenticationEntryPoint: RestAuthenticationEntryPoint,
```

And all the other occurrences.